src/listeners/QuoteSystem.hs

```diff
@@ -16,8 +16,19 @@ import Owoifier (owoify)
 import Text.Parsec (anyChar, many1)
 import Utils (devPerms, modPerms, sendMessageChan, sentInServer)
 
+import Control.Monad.Random
```

Comment:
Can change to `System.Random` as this is what we've been using consistency already and the equivalent (but different name) function exists.